src/porepy/fracs/importer.py

```diff
@@ -11,6 +12,56 @@
 
 #------------------------------------------------------------------------------#
 
+def dfm_from_csv(file_name, tol=1e-4, **mesh_kwargs):
```

Comment:
Naming is critical here. First, I suppose 3d should be somewhere in the name, and that the existing function mesh_from_csv should have 2d in it. Or we want a wrapper function which splits into 2D and 3d, but maybe not.

Then dfm: I know you want to differ from dfn, and I support it. My initial thought is, if specifications are needed, it is dfn that is the special case.

Also, can you split the function into two:  1) Read csv and return FractureNetwork. 2) Given the network, create mesh by calling upon meshing.simplex_grid? Then it will be easy to extend this to dfn if desirable. 

BTW, you don't need to impose boundaries + find and split intersections, this is being taken care of by the meshing functions already.

Reply:
I changed some of the names to be more consistent.
I split the function indicated in two

I don't see where the comment in BTW is in the code. Can you be more specific?